android/app/build.gradle

```diff
@@ -91,6 +91,7 @@ dependencies {
     compile 'com.jakewharton:butterknife:8.4.0'
     compile 'com.amulyakhare:com.amulyakhare.textdrawable:1.0.1'
     apt 'com.jakewharton:butterknife-compiler:8.4.0'
+    compile 'net.opacapp:multiline-collapsingtoolbar:1.3.1'
```

Comment:
Um, no.
Let's not do that.
Try looking for an alternative library and if not then we can merge this instead.
How about you take a look at the I/O app to see how they are doing it.